src/WritePlasterContent.ps1

```diff
@@ -0,0 +1,246 @@
+            }
+            if (-not [string]::IsNullOrEmpty($destination)) {
+                $oXMLContent.SetAttribute("destination", $destination)
+            }
+            if (-not [string]::IsNullOrEmpty($description)) {
+                $oXMLContent.SetAttribute("description", $description)
+            }
+            if (-not [string]::IsNullOrEmpty($author)) {
+                $oXMLContent.SetAttribute("author", $author)
+            }
+            if (-not [string]::IsNullOrEmpty($companyname)) {
+                $oXMLContent.SetAttribute("companyName", $companyname)
+            }
+            if (-not [string]::IsNullOrEmpty($moduleversion)) {
+                $oXMLContent.SetAttribute("moduleVersion", $moduleversion)
+            }
+            if (-not [string]::IsNullOrEmpty($rootmodule)) {
+                $oXMLContent.SetAttribute("rootModule", $rootmodule)
+            }
+            if (-not [string]::IsNullOrEmpty($copyright)) {
+                $oXMLContent.SetAttribute("copyright", $copyright)
+            }
+            if (-not [string]::IsNullOrEmpty($projectUri)) {
+                $oXMLContent.SetAttribute("projectUri", $projectUri)
+            }
+            if (-not [string]::IsNullOrEmpty($licenseUri)) {
+                $oXMLContent.SetAttribute("licenseUri", $licenseUri)
+            }
+            if (-not [string]::IsNullOrEmpty($iconUri)) {
+                $oXMLContent.SetAttribute("iconUri", $iconUri)
+            }
+            if (-not [string]::IsNullOrEmpty($tags)) {
+                $oXMLContent.SetAttribute("tags", $tags)
+            }
+            if (-not [string]::IsNullOrEmpty($encoding)) {
+                $oXMLContent.SetAttribute("encoding", $encoding)
+            }
+            if ($openInEditor) {
+                $oXMLContent.SetAttribute("openInEditor", 'true')
+            }
+
+          }
+          'requireModule' {
+            <#
+              Uses: name, minimumversion, maximumversion, requiredversion, message, condition
+            #>
+            if (-not [string]::IsNullOrEmpty($Condition)) {
+                $oXMLContent.SetAttribute("condition", $Condition)
+            }
+            if (-not [string]::IsNullOrEmpty($Name)) {
+                $oXMLContent.SetAttribute("Name", $Name)
+            }
+            if (-not [string]::IsNullOrEmpty($minimumversion)) {
+                $oXMLContent.SetAttribute("minimumversion", $minimumversion)
+            }
+            if (-not [string]::IsNullOrEmpty($maximumversion)) {
+                $oXMLContent.SetAttribute("maximumversion", $maximumversion)
+            }
+            if (-not [string]::IsNullOrEmpty($requiredversion)) {
+                $oXMLContent.SetAttribute("requiredversion", $requiredversion)
+            }
+            if (-not [string]::IsNullOrEmpty($message)) {
+                $oXMLContent.SetAttribute("message", $message)
+            }
+          }
+        }
+
+        $oXMLRoot.InnerXML
+
+    }
+}
```

Comment:
Please add a newline at the end of the file.  I'm going to add that setting to the VSCode settings file - then it will take care of this issue.